configure.ac

```diff
@@ -928,6 +928,25 @@ if test x$use_upnp != xno; then
     [AC_CHECK_LIB([miniupnpc], [upnpDiscover], [MINIUPNPC_LIBS=-lminiupnpc], [have_miniupnpc=no])],
     [have_miniupnpc=no]
   )
+dnl The minimum supported miniUPnPc API version is set to 10. This keeps compatibility
+dnl with Ubuntu 16.04 LTS and Debian 8 libminiupnpc-dev packages. See details in #15993.
```

Comment:
I don't recommend referring to GitHub issues. I think we're supposed to have a self-contained [set of] git repo(s)?